### PR TITLE
[IMPROVEMENT] Managing DropLists via CLI

### PR DESCRIPTION
**Pull request by Maxxx873**
Hi everyone! I decided to add additional functionality for managing Droplists using the CLI. Also minor refactoring and documentation additions.

### CODE REVIEW

File: docs/modules/servers/partials/operate/webadmin.adoc [hunk 1]
```diff
@@ -2567,6 +2567,67 @@ Response codes:
 
 * 204: The quota has been updated to unlimited value.
 
+===== Domain DropList
```
Comment (Arsnael) [inline docs/modules/servers/partials/operate/webadmin.adoc:2570]: Missing the upper section declaration `Add entry to the DropList`?

Also missing global adding?

File: docs/modules/servers/partials/operate/cli.adoc [hunk 2]
```diff
@@ -272,6 +288,49 @@ level. Note: syntax is similar to what was exposed previously.
...
+* `user`: allow to customize personalized droplist of blocked domains
+and email addresses.
+
+See example commands for `domain` owner scope, `owner@owner.com`
+owner, `evil@devil.com` denied entity.
+
+Add Entry to the DropList
+
+....
+{cli} AddDropListEntry domain owner@owner.com evil@devil.com
```
Comment (Arsnael) [inline docs/modules/servers/partials/operate/cli.adoc:313]: If the example is about the domain, shouldn't the command be then:

```
{cli} AddDropListEntry domain owner.com evil@devil.com
```

Idem for below examples?

File: src/site/markdown/server/manage-cli.md [hunk 3]
```diff
@@ -290,6 +291,45 @@ Moreover, James allows to specify global maximum values, at the server level. No
...
+  Entries in the global owner scope apply universally and affect all users and domains.
+- `domain`: each domain can have its own droplist, which contains entries specific to that domain.
+- `user`: allow to customize personalized droplist of blocked domains and email addresses.
+
+See example commands for `domain` owner scope, `owner@owner.com` owner, `evil@devil.com` denied entity.
+
+Add Entry to the DropList
+
+```
+{cli} AddDropListEntry domain owner@owner.com evil@devil.com
```
Comment (Arsnael) [inline src/site/markdown/server/manage-cli.md:311]: Same remark as above

File: docs/modules/servers/partials/operate/webadmin.adoc (hunk 1 above)
Comment (Arsnael) [reply docs/modules/servers/partials/operate/webadmin.adoc:2570]: Having a pass again for some reason it seems the droplist section aint present in the antora doc at all! I don't know if we missed that when introducing the droplist or if it got lost when we were refactoring the antora doc (if it's the case, sorry)

Then we probably need to add more globally the droplist section as well, as `Getting the DropList` and `Testing a denied entity existence` subsections, to have something similar to https://james.apache.org/server/manage-webadmin.html#administrating-droplists

You think you can help with that @Maxxx873 ?

**Review by Arsnael — COMMENTED** [review]
Looks good to me. Just have some remarks on the doc, but otherwise I'm fine. Thanks again for the contribution @Maxxx873 :)

**Comment by Arsnael** [comment]
I restarted the build by mistake but I think there is related errors in the cli integration tests for droplist: https://ci-builds.apache.org/job/james/job/ApacheJames/job/PR-2454/1/#showFailuresLink

**Comment by Maxxx873** [comment]
Thanks for your comments, @Arsnael! One of the goals of this pr is to improve the documentation, so of course I'll add missing information. And of course I'll be fixing errors in the tests.

**Review by Arsnael — APPROVED** [review]
Thanks for the fixes! LGTM

**Review by quantranhong1999 — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**